tox.ini

```diff
@@ -1,10 +1,11 @@
 # content of: tox.ini , put in same dir as setup.py
 [tox]
-envlist = clean, style_check, py38, doctest
+envlist = clean, style_check, py38, py310, doctest
```

Comment:
https://tox.wiki/en/latest/config.html#conf-skip_missing_interpreters

probably something like.
```
skip_missing_interpreters = true
```

The reason why it fails seems to be due to the fact that the way we set up in github actions has only single python version installed. So we need to skip py310 test for actions with only python 3.8 installed.